test/tests/unit/run/test_acq_manager.py

```diff
@@ -392,7 +392,7 @@ def test_abort_protocol(self):
 
         hfile.close()
 
-    # @unittest.skip("Grrrrrr")
+    @unittest.skip("Grrrrrr")
```

Comment:
It is important to keep an eye on the rep rate timing. Would to prefer to leave this in, or create some other replacement that reports the rates